gbpservice/tests/contrib/nfp_scripts/upgrade_nfp.conf

```diff
@@ -0,0 +1,27 @@
+#---------------------#
```

Comment:
With the introduction of "FROM" and "TO" parameters in the configuration, the name of the conf file is not meaningful. It is no more an upgrade. Lets call this file as mode_shift.conf

Also, this file should go into gbpservice/nfp/config directory. We have to create this new directory.